lib/wallaby/phantom/server.ex

```diff
@@ -25,13 +25,18 @@ defmodule Wallaby.Phantom.Server do
     port = find_available_port()
     local_storage = tmp_local_storage()
 
-    Port.open({:spawn, phantomjs_command(port, local_storage)}, [:binary, :stream, :use_stdio, :exit_status])
+    Port.open({:spawn_executable, script_path()},
+      [:binary, :stream, :use_stdio, :exit_status, args: script_args(port, local_storage)])
 
     {:ok, %{running: false, awaiting_url: [], base_url: "http://localhost:#{port}/", local_storage: local_storage}}
   end
 
-  def phantomjs_command(port, local_storage) do
-    "#{script_path()} #{phantomjs_path()} --webdriver=#{port} --local-storage-path=#{local_storage} #{args()}"
+  def script_args(port, local_storage) do
+    [
+      phantomjs_path(),
```

Comment:
ah, you're right. We pass in the full path to script_path so it shouldn't be an issue.